### ISSUE TITLE
Suggestion: new message style

### ISSUE THREAD

**Issue by Zomblad** [MLG-Fortress/ExtraHardMode#57]
Hello, again XD
I have AMAZING idea for EHM plugin! You really should add a new way of showing messages about new game rules - title messages! You can config that to be message in score board, in chat or in title (and subtitle) (with colors).
Sorry for my awful English, I hope you get the idea :)

**Comment by RoboMWM** [comment]
Titles are highly obtrusive and don't work well for displaying a lot of info (as it's very dependent on the client's GUI settings).

**Comment by Zomblad** [comment]
I hope someday you will add this option. Its easy to change the amount of text. Im asking this becuase TONS of newbie players dont want to read help.

**Comment by RoboMWM** [comment]
Then they will just as easily ignore the title text, even if it is in their face.

If someone wants to put in a PR for this, fine. But I don't see any reason why tutorial messages should show up as a title. Maybe for the "inventory too heavy" notification when drowning, perhaps.

**Comment by Zomblad** [comment]
As you wish )

**Comment by Zomblad** [comment]
tell me if you will make a new update

**Comment by Zomblad** [comment]
Any hopes on adding? (((

**Comment by RoboMWM** [comment]
> If someone wants to put in a PR for this, fine.

Also note that there has yet to be a (proper) Title API in spigot. There is one in paper, but unless I do some additional checks to determine if the server is running paper, it will fail on spigot servers.

**Comment by Zomblad** [comment]
Whats PR?

**Comment by RoboMWM** [comment]
PR: Pull Request

**Comment by Zomblad** [comment]
so, it will be like random?

**Comment by RoboMWM** [comment]
No, it means someone else can feel free to submit code to implement this.

**Comment by RoboMWM** [comment]
Semi-related: #61
